src/k-provider/ovp/index.js

```diff
@@ -9,4 +10,4 @@ declare var __NAME__: string;
 const NAME = __NAME__ + '-ovp';
 const VERSION = __VERSION__;
 
-export {OVPProvider as Provider, NAME, VERSION, RequestBuilder, ResponseTypes};
+export {OVPProvider as Provider, OVPProviderParser, NAME, VERSION, RequestBuilder, ResponseTypes};
```

Comment:
```suggestion
export {OVPProvider as Provider, OVPProviderParser as ProviderParser, NAME, VERSION, RequestBuilder, ResponseTypes};
```

Reply:
What if we want to expose OTTProvider in the future ?

Reply:
ott provider has its own index.js - https://github.com/kaltura/playkit-js-providers/blob/master/src/k-provider/ott/index.js